### PR TITLE
Availability enhancements 

### LINKED ISSUE #1186: Availability status_label values need to be processed
- The values for status_label should be capitalized
- "check_holdings" should be translated to "Some items not available"

https://github.com/pulibrary/bibdata/blob/alma/app/adapters/alma_adapter/availability_status.rb#L34

### LINKED ISSUE #1188: Remove "more_items" key from bibliographic/[:id]/availablility and bibliographic/availability
This key is no longer needed by orangelight, since it was used to let us know to fetch all the availability data for each item, which we can no longer afford to do

### PR DESCRIPTION
**Pull request by hectorcorrea**
Improves the logic for the status_label value and removes a property no longer needed in the response.

Fixes #1186
Fixes #1188

### CODE REVIEW

File: app/adapters/alma_adapter/status.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ def initialize(bib:, holding_item_data:, holding:)
   end
 
   def to_s
-    return "On-Site" if holding["availability"] == "available"
+    return "Some items not available" if holding["availability"] == "check_holdings"
+    return holding["availability"].titlecase if holding["availability"]
+
+    # For electronic holdings
+    return holding["activation_status"].titlecase if holding["activation_status"]
```
Comment (hackartisan) [inline app/adapters/alma_adapter/status.rb:21]: I'm not sure we are aware of this use case from the orangelight side; we might need more info.

**Review by hackartisan — APPROVED** [review]

File: app/adapters/alma_adapter/status.rb (hunk 1 above)
Comment (hectorcorrea) [reply app/adapters/alma_adapter/status.rb:21]: This is the reason I have been hesitant to close the two tickets about availability (single bib and multi-bib). 

There are new cases that Alma brings that we didn't have to deal with in Voyager. Hence the note in the code: https://github.com/pulibrary/bibdata/blob/alma/app/adapters/alma_adapter/availability_status.rb#L28-L64.

Comment (hackartisan) [reply app/adapters/alma_adapter/status.rb:21]: Okay let's talk about it at an alma checkin.

**Comment by hectorcorrea** [comment]
@hackartisan I changed the code to return "Available" rather than "On-Site" and I broke some tests. 

Is there a reason to have both (sometimes Available and some other times On-Site) for the status text? If not, I can update the test to reflect the new status value. Otherwise we'll need to figure out when to return one or the other.

File: app/adapters/alma_adapter/availability_status.rb [hunk 2]
```diff
@@ -73,7 +71,7 @@ def holding_summary(holding)
       holding_item_data = item_data[holding["holding_id"]]
       status = Status.new(bib: bib, holding_item_data: holding_item_data, holding: holding)
       {
-        more_items: holding["total_items"].to_i > 1,
+        # more_items: holding["total_items"].to_i > 1,
```
Comment (eliotjordan) [inline app/adapters/alma_adapter/availability_status.rb:74]: issue: Is this still needed?

**Review by eliotjordan — CHANGES_REQUESTED** [review]
Looks good. Just one comment/change.

File: app/adapters/alma_adapter/availability_status.rb (hunk 2 above)
Comment (hectorcorrea) [reply app/adapters/alma_adapter/availability_status.rb]: Good catch. I've removed it.

**Review by eliotjordan — APPROVED** [review]

**Pull request merged**